packages/components/managed-data-table/src/data-table-settings/data-table-settings.js

```diff
@@ -0,0 +1,189 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import requiredIf from 'react-required-if';
+import { useIntl } from 'react-intl';
+import Constraints from '@commercetools-uikit/constraints';
+import SelectInput from '@commercetools-uikit/select-input';
+import Spacings from '@commercetools-uikit/spacings';
+import DensityManager from './density-manager';
+import ColumnManager from './column-manager';
+import TABLE_SETTINGS_ACTIONS, {
+  COLUMN_MANAGER,
+  DISPLAY_SETTINGS,
+} from '../constants';
+import messages from './messages';
+
+export const getDropdownOptions = ({
```

Comment:
It feels weird to unit test this function. Can't this be achieved with RTL by interacting with the component and selecting the proper UI state?